src/System.Windows.Forms.Primitives/src/System/LocalAppContextSwitches/LocalAppContextSwitches.cs

```diff
@@ -130,5 +134,16 @@ public static bool TrackBarModernRendering
             [MethodImpl(MethodImplOptions.AggressiveInlining)]
             get => GetCachedSwitchValue(TrackBarModernRenderingSwitchName, ref s_trackBarModernRendering);
         }
+
+        /// <summary>
+        ///  Indicates whether certificates are checked against the certificate authority revocation list.
+        ///  If true, revoked certificates will be accepted by WebRequests and WebClients as invalid.
```

Comment:
is this line should be  
```suggestion
        ///  If true, revoked certificates will not be accepted by WebRequests and WebClients as invalid.
```

Reply:
Is this clearer?
"If true, revoked certificates will not be accepted by WebRequests and WebClients as valid."

Reply:
Yep.